timemachine/maps/terminal_bonds.py

```diff
@@ -0,0 +1,244 @@
+"""Summarize HarmonicBonds as Intervals in R+, construct maps between Intervals, & apply these maps to bond lengths"""
```

Comment:
Can you add documentation to clarify when this can be used?

Reply:
Added in https://github.com/proteneer/timemachine/pull/801/commits/5db990247755a73f4b328962114a9960a862c4f4